### DISCUSSION TITLE
SV06+ - Moonraker shows this now ?

### DISCUSSION THREAD

**Discussion by Manfredo1952** [bassamanator/Sovol-SV06-firmware#88] [Q&A]
![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/56330568/e7a88e94-c800-4f5e-9283-98b97a77a907)

Printer/printing does work well. Just wonder whether / how to cure this ?

**Comment by bassamanator** [comment]
Have you installed a camera related plugin?

I imagine if you:
```sh
cd ~/moonraker
git status
```
it will list the same file under `Untracked files`?

**Comment by Manfredo1952** [comment]
Thanks. Installed timelaps for unknown reasons   :-)   -    based on your suggestion I have now added the file mentioned to be tracked. Done  👍

**Comment by bassamanator** [comment]
It's not as simple as that. Next time you try to update Moonraker, it will fail. It's best to simply leave the file untracted. Just do a `git reset` to unstage the file. But it's up to you.
